ui/assets/scripts/editor-utils.mjs

```diff
@@ -1,4 +1,5 @@
 import uuidV4 from 'uuid/v4.js';
+import Channel from '~~/lib/model/Channel.mjs';
```

Comment:
Importing the whole Channel class in the fixture editor (and adding it to all the components' data) just for a better readability seems to be overkill, as we should keep the file sizes for client side scripts as small as possible. Maybe add the constants to this file instead?